test/winter_onboarding_2021/fleet_management/healthcheck_test.clj

```diff
@@ -11,6 +11,6 @@
   (testing "Healthcheck for DB"
     (is (= {:status 200
             :headers {}
-            :body {:message "Everything OK!"
-                   :data [{:check 1}]}}
+            :body (str {:message "Everything OK!"
```

Comment:
Why is the handler returning a string?

Reply:
We removed the `wrap-json-body` middleware and without that, `ring` can not handle hashmaps.

Reply:
I will wrap the healthcheck route to use json